client/allocrunner/taskrunner/task_runner.go

```diff
@@ -364,6 +364,13 @@ func (tr *TaskRunner) initLabels() {
 		},
 	}
 
+	if tr.alloc.Namespace != "" {
```

Comment:
Can this be actually empty?  If not, i'd inline it above.

Reply:
correct, this field is normalized to "default"